test/integration/kubernetes_deploy_test.rb

```diff
@@ -619,7 +619,7 @@ def test_failure_logs_from_unmanaged_pod_appear_in_summary_section
 
     assert_logs_match_all([
       "Failed to deploy 1 priority resource",
-      "Logs from container 'hello-cloud' (last 250 lines shown):",
+      "Logs from container 'hello-cloud':",
```

Comment:
why did this change?

Reply:
It was irrelevant in cases like this (there's only one line here), and in other cases the message was lying: [here](https://github.com/Shopify/kubernetes-deploy/pull/340/files#diff-e6b2b670631bf178ac256ed1780cf5a4L97) it conditionally truncated the number of lines requested, but [here](https://github.com/Shopify/kubernetes-deploy/pull/340/files#diff-d2654750695bbb60cf39d64df9b868eeL225) it unconditionally printed that they were truncated.

Keeping the code like that made even less sense now that the possible truncation is happening in another class entirely, and by slightly different logic. The new logic is that if this is our first request, tail 250 lines; otherwise, ask for all lines since our last request (based on timestamp). So I only print this message if there are exactly 250 lines now, [here](https://github.com/Shopify/kubernetes-deploy/pull/340/files#diff-d2654750695bbb60cf39d64df9b868eeR229).

Incidentally I think 250 is way too high (we picked it kinda arbitrarily to begin with), but that's not really related to this PR.